src/Orckestra.Composer.Website/App_Data/Razor/Composer/Search/Summary.cshtml

```diff
@@ -34,7 +34,8 @@
 <head>
 </head>
 <body>
-    <div class="island" data-oc-controller="Product.SearchSummary" id="vueSearchSummary" data-context="@JsonConvert.SerializeObject(ProductsSearchViewModel.ProductSearchResults)" v-cloak="true">
+    <div class="island" data-oc-controller="Product.SearchSummary" id="vueSearchSummary" 
```

Comment:
can we use separate Browsing.SearchSummary  to avoid this confusing with Tabs which are absent here